index.html

```diff
@@ -3,12 +3,17 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="./modules/dateTime.js" defer type="module"></script>
     <script src="./modules/index.js" defer type="module"></script>
     <script src="./modules/list.js" defer type="module"></script>
     <script src="./modules/contact.js" defer type="module"></script>
     <script src="./modules/add.js" defer type="module"></script>
+    <script
+      src="./node_modules/luxon/build/es6/luxon.js"
+      type="module"
+    ></script>
     <link rel="stylesheet" href="styles.css" />
```

Comment:
- [ ] [OPTIONAL] It would be nice if you implement same wireframe as original 👍🏼 

![image](https://github.com/enocol/AwesomeBooks/assets/63915024/163bee82-713e-4719-a15b-f94d3b19dd50)